### PR TITLE
initial fastlane structures

### PR DESCRIPTION
**Pull request by IzzySoft**
The promised "Fastlane starter kit" :wink:

### CODE REVIEW

**Comment by jgaa** [comment]
Thanks! I appreciate it.

**Comment by IzzySoft** [comment]
Gladly! I had it setup anyway – just needed to reformat so F-Droid doesn't break formatting. The process there replaces each and every `\n` by `<br>\n` without applying much other formatting – I use HTML without that `nl2br`; so I converted it to "simple Markdown with simple inline HTML for emphasis", which F-Droid can work with quite fine – while I'm now telling my updater "that's Markdown over there, convert it **properly** when you fetch it"; which again results in good and proper HTML, looking better formatted than "there" :wink:

**TL;DR:** the formatting from this PR should work fine with F-Droid, my repo, and Google Play (if needed; but the latter most likely requires the Fastlane binaries).
